File: C#/3/3_2.cs
const double NON_MEMBER_DISCOUNT = 0.95; // 5% discount for non-members
const double MEMBER_DISCOUNT = 0.90;    // 10% discount for members

double finalPrice = basePrice * NON_MEMBER_DISCOUNT;
if (isMember)
{
    finalPrice = basePrice * MEMBER_DISCOUNT;
}

Console.WriteLine("Final price: $" + finalPrice);

File: JavaScript/3/3_2.js
const everyoneDiscount = 0.95;
const membershipDiscount = 0.90;

let finalPrice = basePrice * everyoneDiscount; // Apply 5% discount
if (isMember) {
    finalPrice = basePrice * membershipDiscount; // Members get 10% discount
}
console.log("Final price: $" + finalPrice);
